test_scripts/Defects/7_1/1585_AlertManeuver_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE resultCode to Navigation.AlertManeuver component
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session started
+-- 3) App is registered
+-- 4) App is activated
+--
+-- Steps:
+-- 1) Send AlertManeuver mobile RPC from app with parameter imageType = "STATIC", HMI responds with
+--    UNSUPPORTED_RESOURCE to Navigation.AlertManeuver request
+--
+-- Expected:
```

Comment:
Please update expected like in https://github.com/LuxoftSDL/sdl_atf_test_scripts/blob/master/test_scripts/Defects/7_1/2604_Starting_of_protected_RPC_service_before_and_after_app_registration.lua
`SDL does`